src/storage/mongodb/TransactionStorage.ts

```diff
@@ -93,6 +93,7 @@ export default class TransactionStorage {
       currentInstantAmpsL2: Utils.convertToInt(transactionToSave.currentInstantAmpsL2),
       currentInstantAmpsL3: Utils.convertToInt(transactionToSave.currentInstantAmpsL3),
       currentInstantAmpsDC: Utils.convertToInt(transactionToSave.currentInstantAmpsDC),
+      phasesUsed: transactionToSave.phasesUsed,
```

Comment:
Do not assign a whole json document but rather each property with the right usage of Utils methods to convert to the right type.